bitcoin/examples/taproot-psbt.rs

```diff
@@ -412,7 +412,7 @@ impl BenefactorWallet {
             taproot_spend_info.internal_key(),
             taproot_spend_info.merkle_root(),
         );
-        let value = input_utxo.amount - ABSOLUTE_FEES;
+        let value = (input_utxo.amount - ABSOLUTE_FEES).unwrap();
```

Comment:
In 4d6911b12c0dec91ebf2ba1065224300e6231ad5:

Since this is example code we should do `expect("ABSOLUTE_FEES must be set below input amount")` or something. This feels like the kind of error path we'd actually trigger when messing with the example and it'd be nice to have some clear feedback. (Previously I think we'd get a panic which explicitly fingered numeric overflow, and in release mode it'd just do something nonsensical.)

Reply:
Used as suggested (in two places).